flask/migrations/versions/6f448fc94a2d_many_to_many_dataset_files.py

```diff
@@ -8,7 +8,8 @@
 from alembic import op
 import sqlalchemy as sa
 
-from app.extensions import db
+# NOTE: DO NOT DO THIS, STATIC MIGRATIONS CANNOT DEPEND ON A CHANGING MODEL FILE
```

Comment:
I take it that this will be left as a future issue?

Reply:
Yeah, I actually noticed this way back when I removed the metadatasets table and it broke a migration that was added while I was away. It's a _fait accompli_ at this point, but we do need to keep note of this as in development environments, we are running all the migrations from the very first one, instead of just any new ones that were added.